jerry-core/include/jerryscript-compiler.h

```diff
@@ -144,6 +139,16 @@ void * __cdecl _alloca (size_t _Size);
 #define JERRY_ATTR_WARN_UNUSED_RESULT
 #endif /* !JERRY_ATTR_WARN_UNUSED_RESULT */
 
+/**
+ * Macro to setup function attributes to force inline function to all call sites.
+ * Before it's usage `static` or `extern` modifier must be used.
+ * Because in msvc, inline without `extern` would cause the function don't generate
+ * symbol and then cause linkage error.
+ */
+#ifndef JERRY_ALWAYS_INLINE
+#define JERRY_ALWAYS_INLINE(x) inline
```

Comment:
This is not a function-like macro anymore.

Reply:
OK